examples/robolectricExample/src/test/java/io/realm/examples/robolectric/TestHelper.java

```diff
@@ -0,0 +1,39 @@
+package io.realm.examples.robolectric;
```

Comment:
I would remove this file. It doesn't really provide much in this example project